packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/v1/aio/operations/_multiapi_service_client_operations.py

```diff
@@ -148,21 +148,26 @@ async def _test_lro_initial(
         _request = _convert_request(_request)
         _request.url = self._client.format_url(_request.url)
 
-        _stream = False
+        _stream = kwargs.pop("stream", False)
         pipeline_response: PipelineResponse = await self._client._pipeline.run(  # type: ignore # pylint: disable=protected-access
             _request, stream=_stream, **kwargs
         )
 
         response = pipeline_response.http_response
 
         if response.status_code not in [200, 204]:
+            if _stream:
+                await response.load_body()  # Load the body in memory and close the socket
```

Comment:
see 4 of https://github.com/Azure/autorest.python/pull/2589#issue-2296719842. I infer it may be related with https://github.com/Azure/azure-sdk-for-python/blob/3930c7cb4ddf40610a31115b9258cf00e9d53451/sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py#L512

Reply:
We shouldn't use `load_body` since it's a deprecated method. You should call `.read()` instead

Reply:
see 4 and 5 of https://github.com/Azure/autorest.python/pull/2589#issue-2296719842. legacy HttpResponse type doesn't have `.read()`